.github/docker/Dockerfile.production

```diff
@@ -1,37 +0,0 @@
-FROM debian:bookworm-slim
-
-ARG DEB_FILE
-ARG DEB_DEBUG_FILE
-
-ENV DEBIAN_FRONTEND=noninteractive
-
-USER root
-
-RUN apt-get update && apt-get install adduser && apt-get clean
-
-RUN addgroup --gid 10001 --system liquidsoap && \
-    adduser --system --disabled-password --disabled-login --uid 10000 \
-    --home /var/cache/liquidsoap --ingroup liquidsoap liquidsoap && \
-    usermod --append --groups audio liquidsoap
```

Comment:
@toots,
Why is the user `10000` and `system`, and the group is `10001` and `system`?

Reply:
No idea. These have been copy/pasted for a while. We should prolly remove the explicit IDs

Reply:
Also, are there other groups we should add the user to?

Reply:
I removed the `groupadd` and `useradd` commands in the docker container, so the new settings are determined by the script in the `.deb` package.
https://github.com/savonet/liquidsoap/blob/cde4dad5a3156540fe2577f68ecbe42f663da097/.github/debian/postinst#L22-L27